docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,181 @@
+# Multinomial {#openvino_docs_ops_generation_Multinomial_13}
+
+@sphinxdirective
+
+.. meta::
+  :description: Learn about Multinomial-13 - a generation operation, that creates a sequence of indices of sampled classes from Multinomial distribution.
+
+**Versioned name**: *Multinomial-13*
+
+**Category**: *Generation*
+
+**Short description**: *Multinomial* operation generates a sequence of class indices sampled from Multinomial distribution.
+
+**Detailed description**: *Multinomial* operation generates a sequence of class indices sampled from a Multinomial distribution. In this context, the *input* values represent the probabilities associated with each class within the Multinomial distribution. When performing the operation, it randomly selects a class based on these probabilities. Subsequently, the index of the chosen class in the *input* array is appended to the *output* sequence in the corresponding batch.
+
+**Algorithm formulation**:
+
+Given a list of probabilities x1,x2, ..., xn:
+* If *log_probs* is true:
+  * For each probability x, replace it with a value e^x
+* Create a cumulative sum of those probabilities, ie. create an array of values where the ith value is the sum of the probabilities x1, ..., xi
+* Divide the created array by its last (maximum) value to normalize the cumulative probabilities between [0, 1]
+* Randomly generate a sequence of floats in the range range [0,1]
+* For each generated number, assign the class with the lowest index for which the cumulative probability is less or equal the generated value.
+* If *with_replacement* is False (sampling without replacement)
+  * Set the class probability to 0 by adjusting the cumulative sum
+  * Assume class with index i has been selected - then every cdf value starting at i-th index should be lowered by the probability of the selected class
+  * Afterwards, divide the cdf by its last (maximum) value to normalize the cumulative probabilities between [0, 1]
+* Convert the output indices to *output_type*
+* Return output indices
+
+**Example computations**:
+Example 1 - 1D tensor
+* Let *input* = ``[0.1, 0.5, 0.4]``, *num_samples* = 5, *log_probs* = false, *replacement* = true
+* CDF of *input* = ``[0.1, 0.1 + 0.5, 0.1 + 0.5 + 0.4]`` = ``[0.1, 0.6, 1]``
```

Comment:
Is there any description of this acronym `CDF`?
If not can you add it (also for any other used in this document), or add link where they are described.